mne/gui/_coreg_gui.py

```diff
@@ -1282,17 +1333,23 @@ def __init__(self, raw=None, subject=None, subjects_dir=None,
                         "(run $ mne make_scalp_surfaces).")
                 raise ValueError(msg)
             self.model.mri.subject = subject
+        if trans is not None:
+            self.model.load_trans(trans)
+
+    @on_trait_change('subject_panel.subject')
+    def _set_title(self):
+        self.title = '%s - MNE Coreg' % self.model.mri.subject
 
     @on_trait_change('scene.activated')
     def _init_plot(self):
-        self.scene.disable_render = True
+        _toggle_mlab_render(self, False)
```

Comment:
Yeah probably. But let's do it in another PR when someone is bored so we don't break existing PRs.